client/me/help/chat-business-concierge-notice/index.jsx

```diff
@@ -19,27 +17,26 @@ import { isBusinessPlanUser } from 'state/selectors';
 
 class ChatBusinessConciergeNotice extends Component {
 	static propTypes = {
-		translate: PropTypes.func,
-		isBusinessPlanUser: PropTypes.bool.isRequired,
 		from: PropTypes.string.isRequired,
 		selectedSite: PropTypes.object.isRequired,
 		to: PropTypes.string.isRequired,
-	};
 
-	static defaultProps = {
-		translate: identity,
+		// Connected props
+		isBusinessPlanUser: PropTypes.bool.isRequired,
+		moment: PropTypes.func.isRequired,
+		translate: PropTypes.func.isRequired,
```

Comment:
Diff is a bit ugly, but `defaultProps` which only included `translate` is removed and `propTypes` are organized.